CreateSigningDirectory.cmd

```diff
@@ -25,21 +25,32 @@ call :CreateDirIfNotExist %SigningDirectory%
 call :CreateDirIfNotExist %SigningDirectory%\net461\
 call :CreateDirIfNotExist %SigningDirectory%\netcoreapp2.0
 
-call :CopyFilesForMultitargeting Json.Schema.dll            || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Pointer.dll           || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
-call :CopyFilesForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.dll            || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Pointer.dll           || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.ToDotNet.dll   || goto :ExitFailed
+call :CopyDllForMultitargeting Json.Schema.Validation.dll || goto :ExitFailed
+
+call :CopyExeForMultitargeting Json.Schema.ToDotNet.Cli   || goto :ExitFailed
+call :CopyExeForMultitargeting Json.Schema.Validation.Cli || goto :ExitFailed
 
 goto :Exit
 
-:CopyFilesForMultitargeting
+:CopyDllForMultitargeting
 xcopy /Y %BinaryOutputDirectory%\%~n1\netstandard2.0\Microsoft.%1  %SigningDirectory%\netcoreapp2.0\
 if "%ERRORLEVEL%" NEQ "0" (echo %1 assembly copy failed. && goto :CopyFilesExit)
```

Comment:
>CopyFilesExit [](start = 66, length = 13)

This label no longer exists. Did you mean to move it down to just above Line 54, rather than just removing it?